templates/moduleEdit.html

```diff
@@ -13,54 +13,32 @@
 <body>
     <div class="container-block container">
         <div class="row red">
-            <div class="col-md-6 module-info-wrapper">
-                <h1 class="row title text-center">Edit: $moduleCode</h1>
+            <div class="col-md-12">
                 <div class="row">
-                    <!--module information-->
-                    <form id="edit-module-form" name="edit-module-form" action="/editModule" method="post">
-                        <input type="hidden" name="code" value="$moduleInfo[0]" required>
-                        <div class="form-group">
-                            <label for="module-name">Module Name:</label>
-                            <input type="text" name="name" value="$moduleName" class="form-control" id="module-name" pattern="[a-zA-Z0-9\s]+" title="Module name should be alphanumeric." required>
-                        </div>
-                        <div class="form-group">
-                            <label for="no-mc">Number of MCs</label>
-                            <input type="number" min="0" max="12" size="10" name="mc" value="$moduleMC" class="form-control" id="no-mc" placeholder="no. MCs" required>
-                        </div>
-                        <div class="form-group">
-                            <label for="description">Description</label>
-                            <textarea type="text" name="desc" class="form-control" rows="6" id="description" placeholder="description">$moduleDescription</textarea>
-                        </div>
-                        <input type="hidden" name="status" value="submit" required>
-                        <input class="btn btn-primary" type="submit" value="Submit" />
-                    </form>
+                    <h1 class="row title text-center">Edit: $moduleCode</h1>
```

Comment:
This is <code>Edit Module</code>:

![image](https://cloud.githubusercontent.com/assets/12386860/24054453/40ba7a56-0b77-11e7-8973-a2fefb665672.png)

Now this is <code>Edit Mounting</code>:

![image](https://cloud.githubusercontent.com/assets/12386860/24054471/547dcad4-0b77-11e7-82fe-d7c9068fd8cc.png)

**Title style is inconsistent, please fix.**